docs/output.md

```diff
@@ -2,67 +2,286 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
-
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
+The directories listed below will be created in the results directory after the pipeline has finished.
+All paths are relative to the top-level results directory.
 
 ## Pipeline overview
 
-The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
+The pipeline is built using [Nextflow](https://www.nextflow.io/) and the results are organized as follow:
+
+- [Original output](#original-output)
```

Comment:
"original output" means the output from the programs that don't get any modification. So for instance, the output of Prokka module is the same as the output of the program itself. While we have some "custom metatdenovo output" that are different from the output of the programs.
About the heading I wanted in that way because there are two sections "Original output" and "custom metatdenovo output" then under "Original output" there are other sections as "preprocessing" which contains several bullet points. I don't know if I was clear, it might sounds a bit confusing :)

Reply:
Ah, this is a table of contents.

The term "original output" is confusing as original can mean many things such as novel, first, etc. Perhaps use something like "module output" or "software output" instead.

Reply:
OK, I will change it then!